fe/fe-core/src/main/java/org/apache/doris/task/CreateReplicaTask.java

```
@@ -187,7 +189,14 @@ public void setBaseTablet(long baseTabletId, int baseSchemaHash) {
     }
 
     public void setStorageFormat(TStorageFormat storageFormat) {
-        this.storageFormat = storageFormat;
+    	if (storageFormat == TStorageFormat.V1) {
+    		// Ignore v1 format, the storage format must not be v1
+    		// V1 will be removed in the future
+    		LOG.error("StorageFormat.V1 is not supported any more, will use V2 automatically");
```

Throw exception instead of print error log.
No one will notice this log.